docs/GEMINI.md

```
@@ -0,0 +1,23 @@
+# The Weaver's Guide (`packages/looms`)
+
+This document describes how the AI "Weaver" generates the soul of the realm.
+
+## The Seed Phrase
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)